modules/node-group/providers.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.14"
+  required_version = ">= 1.3.0"
```

Comment:
Is this relevant for the prevent_destroy? If not, I would higly suggest to do this in an own PR, to make it visible.

Reply:
No, we can make it in a separate PR. Will remove it from here.